apps/public/src/app/(map)/locatie/kies/_components/SideBar/SideBar.tsx

```diff
@@ -40,10 +94,33 @@ export const SideBar = ({ coordinates }: Props) => {
         </Paragraph>
       </div>
       <div>
-        <Heading level={2} size="level-4">
-          Zoek op adres
-        </Heading>
-        {address && <div>{address}</div>}
+        <Field>
+          <HUILabel as={Label}>Zoek op adres</HUILabel>
+          <Combobox value={address} onChange={setAddress} onClose={() => fetchAddressList('')}>
+            <ComboboxInput
```

Comment:
@vdegraaf wrote: The voice over says you can press cntrl-option-space to see option. But this doesn't work. Is this behaviour we can adjust? (see screenshot)

Hm, that's odd. It's something I can't really test. Do you encounter the same issue [here](https://alphagov.github.io/accessible-autocomplete/examples/) and [here](https://headlessui.com/react/combobox)?

Reply:
Nope, there I'm getting this:

<img width="752" alt="Screenshot 2025-01-21 at 13 08 35" src="https://github.com/user-attachments/assets/9dba7edf-f0c2-4008-be50-501dd79017d0" />

Reply:
I think I fixed this, can you check?